test/e2e/util.go

```diff
@@ -316,6 +317,17 @@ var grafanaTestCases = []struct {
 			},
 		},
 	},
+	{
+		dashboardName: "network_topology_dashboard",
+		dashboardUid:  networkTopologyDashboardUid,
+		queryList: []query{
+			{
+				queryId:      0,
+				panelName:    "Network Topology",
+				expectResult: []string{controlPlaneNodeName(), workerNodeName(1), toExternalClientName, toExternalServerIP},
```

Comment:
1. I'm confused. Why do `toExternalClientName, toExternalServerIP` appear in the response string? Doesn't the panel query rule out to-external traffic by requiring "destinationPodName" non-empty?
2. could we add more pod and service names to check? e.g. "perftest-a", "perftest-b", "perftest-c", "perftest-d", "perftest-e", "antrea-test/perftest-b", "antrea-test/perftest-c"